<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardPageDeclProperties</class>
 <widget class="QWizardPage" name="WizardPageDeclProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Данные декларации</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="4" column="0">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>Признак формы отчетности</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Отчетный квартал</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="declCorrNumber"/>
     </item>
     <item row="2" column="3">
      <widget class="QDateEdit" name="declYear">
       <property name="displayFormat">
        <string>yyyy</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Вид декларации</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QComboBox" name="declType"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="declDocType">
       <item>
        <property name="text">
         <string>Первичная</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Корректирующая</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Вид документа</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Номер корректировки</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Год</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="3">
      <widget class="QComboBox" name="declSign">
       <item>
        <property name="text">
         <string>Производство, хранение и поставки произведенного этилового спирта питьевого</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Производство, хранение и поставки произведенных спиртных напитков</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Производство, хранение и поставки произведенных вин</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Другой вид деятельности</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>Предоставляется в</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QLineEdit" name="declProvidedFor">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="declQuarter">
       <item>
        <property name="text">
         <string>1 квартал</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 квартал</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 квартал</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4 квартал</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>declType</tabstop>
  <tabstop>declDocType</tabstop>
  <tabstop>declCorrNumber</tabstop>
  <tabstop>declYear</tabstop>
  <tabstop>declProvidedFor</tabstop>
  <tabstop>declSign</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
